soweego/target_selection/musicbrainz/musicbrainz_baseline_matcher.py

```diff
@@ -28,32 +29,18 @@ def get_musicbrainz_artists_from_dump(opened_file_dump, label_column_index, id_c
 
     return label_musicbrainz
 
-def get_path():
-    """Return the path of the current module"""
-    path = os.path.abspath(__file__)
-    return os.path.dirname(path)
-
-def get_output_path():
-    """Returns the path of the output files"""
-    output_dir_path = '%s/output' % get_path()
-
-    if not os.path.exists(output_dir_path):
-        os.makedirs(output_dir_path)
-
-    return output_dir_path
-
 def get_label_musicbrainzid_dict():
     """Returns the name-musicbrainzid dictionary. If it's not stored, creates a brand new file"""
-    filepath = '%s/artists.json' % get_output_path()
+    filepath = '%s/artists.json' % common.get_output_path()
```

Comment:
Please move hardcoded I/O files to constant declarations on top of the source code.